extension/package.json

```diff
@@ -1,21 +1,21 @@
 {
-  "name": "sparkmonitor-js",
+  "name": "jupyterlab_sparkmonitor",
   "version": "0.0.1",
-  "description": "Jupyter Notebook extension to monitor Apache Spark Jobs",
+  "description": "Jupyter Lab extension to monitor Apache Spark Jobs",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/krishnan-r/sparkmonitor.git"
+    "url": "git+https://github.com/itsjafer/sparkmonitor.git"
```

Comment:
In my email exchange with the original author, he mentioned he's cool with a new/separate repo specifically for the lab extension since the notebook and lab extensions are not cross-compatible so I was planning to just maintain this repo as the jupyterlab version of the extension.